src/components/Form/index.tsx

```diff
@@ -50,7 +51,7 @@ export interface FormValidationError {
   error: React.ReactChild;
 }
 
-export interface FormProps<FormValues> extends StandardProps {
+export interface FormProps<FormValues> extends StandardProps, ReactComponentDefaultProps {
```

Comment:
Why not add these to StandardProps?

Reply:
Will conflict with other interfaces that extend from them. 

For example, `children: void` is set in one of them.